src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.h

```diff
@@ -30,37 +31,45 @@ struct grpc_s2a_credentials_options {
  public:
   ~grpc_s2a_credentials_options();
 
-  /** Setter methods. They do not take ownership of the arguments, nor do they
-   * remove a duplicate ciphersuite or target service account from the
-   * appropriate vector, if a duplicate exists. **/
-  /** The setter method for |handshaker_service_url_|. This does not take
-   *  ownership of the argument. **/
-  void set_handshaker_service_url(const char* handshaker_service_url);
+  /** Getters for member fields. **/
+  const std::string HandshakerServiceUrl() const {
```

Comment:
C++ style guide mentions - "For accessors and mutators, it may be named like variables" which I believe is the convention adhered by gRPC C-core as well. So I would rename this API as `handshaker_service_url()`. Sorry for the confusion.

Reply:
No worries - changed the accessors and mutators back to the C naming style.